File: Python/백준/Bronze/3449. 해밍 거리/해밍 거리.py
t = int(input())
for i in range(t):
    s1 = input()
    s2 = input()
    cnt = 0
    for i in range(len(s1)):
        if (s1[i]!=s2[i]):
            cnt+=1
    print(f"Hamming distance is {cnt}.")

File: C#/백준/Bronze/3449. 해밍 거리/해밍 거리.cs
using System;

namespace q{
    class w{
        static void Main(String[] args){
            int a = int.Parse(Console.ReadLine());
            for (int i=0;i<a;i++){
                string x = Console.ReadLine();
                string y = Console.ReadLine();
                int cnt = 0;
                for (int j=0;j<x.Length;j++){
                    if (x[j]!=y[j]){
                        cnt++;
                    }
                }
                Console.WriteLine($"Hamming distance is {cnt}.");
            }
        }
    }
}